csrc/quantization/machete/generate.py

```diff
@@ -166,32 +252,34 @@ class ScheduleConfig:
     tile_scheduler: TileSchedulerType
 
 
-@dataclass
+@dataclass(frozen=True)
 class TypeConfig:
-    element_a: DataType
-    element_b: Union[DataType, VLLMDataType]
-    element_b_scale: DataType
-    element_b_zeropoint: DataType
-    element_d: DataType
+    a: DataType
+    b: Union[DataType, VLLMDataType]
+    b_group_scale: DataType
+    b_group_zeropoint: DataType
+    b_channel_scale: DataType
+    b_token_scale: DataType
```

Comment:
If B is still the weights, shouldn't this be `a_token_scale`?

Reply:
yup, missed renaming, good catch! thanks!

Done